tutorials/lodash/truncate.md

```diff
@@ -0,0 +1,38 @@
+The `truncate` function allows you to shorten a string if it is longer than a specifed length.
+It will shorten the string and replace the last characters of the shortened string with the characters you specify, or the default `...`.
+It takes three options:
+
+- length (number), the maximum length of the string.
+- omission (string), the string to indicate text is omitted.
+- separator (RegExp | string), the separator pattern to truncate to. 
+Meaning that it will cut the string at the indicated pattern/string if the max length is smaller than the string but greater than the position of the indicated pattern/string.
+
+```javascript
+const _ = require('lodash');
+
+const example = 'Masteringjs.io is a great website to learn JavaScript fundamentals, mongoose, vue, and other fun JavaScript libraries.';
+
+const result = _.truncate(example, {
```

Comment:
Create separate examples with

1. Just `length` option. No `omission` or `separator`
2. `length` and `omission`. No `separator`